Document/0x06b-Basic-Security-Testing.md

```diff
@@ -327,15 +327,40 @@ Keychain Data: WOg1DfuH
 
 Note however that this binary is signed with a self-signed certificate with a "wildcard" entitlement, granting access to *all* items in the Keychain - if you are paranoid, or have highly sensitive private data on your test device, you might want to build the tool from source and manually sign the appropriate entitlements into your build - instructions for doing this are available in the GitHub repository.
 
-<!--
-
 ##### Security Profiling with Introspy
 
-Intospy is an open-source security profiler for iOS released by iSecPartners. Built on top of substrate, it can be used to log security-sensitive API calls on a jailbroken device.  The recorded API calls sent to the console and written to a database file, which can then be converted into an HTML report using Introspy-Analyzer <code>[32]</code>.
+Introspy is an open-source security profiler for iOS released by iSecPartners. Built on top of Substrate, it can be used to log security-sensitive API calls on a jailbroken device. The recorded API calls sent to the console and written to a database file, which can then be converted into an HTML report using Introspy-Analyzer. 
+
+After successful installation of IntroSpy, respring the iOS device and follow the below steps: 
+
+* Go to Settings and you will see a section for Introspy.
+* There should be two new menus in the device Settings. 
+	* Instrospy - Apps: The Apps menu allows you to select which applications will be profiled 
+	* Instrospy - Settings: The Settings menu defines which API groups are being hooked
+* Now, kill and restart the app you want to monitor 
+* Go to General > Settings > Introspy - Apps > Select the target app
+
+Once the target app has been selected, make sure it is running. If it is running, quit it and restart the app again. Make sure that your device is connected to your computer as we want to see the device logs that the Introspy analyzer will be logging. Also, open Xcode on your machine, go to Window -> Devices. Choose your device from the menu on the left and select Console. You will now be able to see the device logs while browsing the application. Please note that analyzer will work in the background to collect the information.
+
+###### Generating HTML Reports with Introspy 
+
+* The tracer will store data about API calls made by applications in a database stored on the device (actually one in each folder). This database can be fed to a Python script call Introspy-Analyzer in order to generate HTML reports that make it a lot easier to review the data collected by the tracer. The script will also analyze and flag dangerous API calls in order to facilitate the process of identifying vulnerabilities within iOS applications. 
```

Comment:
" This database can be fed to a Python script call" -> "This database can be fed to a Python script called"